devDocs/startupShutdown.md

```diff
@@ -74,14 +74,19 @@ Steps:
 ### Running the *.exe
 
 Steps:
- 1. Run the NVDA exe from cmd
+ 1. Press `win+r`
+ 1. Enter <path to nvda.exe>
+ 1. Press enter
  1. Observe NVDA starts
 
 Variation:
-- installed copy, portable copy, or installer
+- using an installer .exe, an installed copy .exe, or a portable copy .exe

```

Comment:
With an installed copy you can just type `nvda`. 

For the installer and portable copy an example would be helpful. A common question might be which exe? In a portable copy there are:
- `nvda.exe`
- `nvda_eoaProxy.exe`
- `nvda_noUIAccess.exe`
- `nvda_slave.exe`
- `nvda_uiAccess.exe`

For the installer perhaps add "(launcher)" since it also gets called that, and a common example path:
- `C:\Users\username\Downloads\nvda_2021.1.exe`

Reply:
```suggestion
- using an installer (launcher)
   -  eg: `C:\Users\username\Downloads\nvda_2021.1.exe`
- using an installed copy
   - just type `nvda` in place of the .exe
- using a portable copy
   - find and use the path to `nvda.exe`, located within the portable copy directory
```